crates/bevy_math/src/ray.rs

```diff
@@ -52,13 +52,13 @@ mod test {
         assert_eq!(Some(1.), ray.intersect_plane(Vec3::Z, Vec3::ONE));
         assert_eq!(None, ray.intersect_plane(Vec3::NEG_Z, Vec3::ONE));
 
-        // Parralel
+        // Parallel
         assert_eq!(None, ray.intersect_plane(Vec3::X, Vec3::X));
 
-        // Parralel with simulated rounding error
+        // Parallel with simulated rounding error
         assert_eq!(
             None,
-            ray.intersect_plane(Vec3::X, Vec3::X + Vec3::Z * f32::EPSILON)
+            ray.intersect_plane(Vec3::X, Vec3::new(1., 0., f32::EPSILON))
```

Comment:
I find the line before to be more readable

Reply:
+1 here: the previous test better explains intent.